docs/user-guide/admin-cli.md

```diff
@@ -390,8 +390,8 @@ usage: vmdkops_admin.py tenant access set [-h]
                                           [--volume-totalsize Num{MB,GB,TB} - e.g. 2TB]
                                           --name NAME
                                           [--volume-maxsize Num{MB,GB,TB} - e.g. 2TB]
-                                          [--allow-create] --datastore
-                                          DATASTORE
+                                          [--allow-create ALLOW_CREATE]
```

Comment:
As we discussed, we will address the change to make add and set commands consistent separately.